### PR TITLE
Add spider for asics_eu

### PR DESCRIPTION
**Pull request by abbensidz**

### CODE REVIEW

File: locations/spiders/asics_de.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class AsicsDeSpider(scrapy.Spider):
+
+    name = "asics_de"
+    item_attributes = {"brand": "asics", "brand_wikidata": "Q327247"}
+    allowed_domains = ["www.asics.com"]
+    start_urls = ["https://cdn.crobox.io/content/ujf067/stores.json"]
+
+    def parse(self, response):
+        data_json = json.loads(response.text)
+        for store in data_json:
```
Comment (Cj-Malone) [inline locations/spiders/asics_de.py:18]:
```suggestion
        for store in response.json():
```

File: locations/spiders/asics_de.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            item = GeojsonPointItem()
+            item = DictParser.parse(store)
```
Comment (Cj-Malone) [inline locations/spiders/asics_de.py:20]: `DictParser.parse` creates the item for us.

```suggestion
            item = DictParser.parse(store)
```

**Comment by Cj-Malone** [comment]
I don't know if all of these are Asics stores, `storetype`:

```
    120 "expert-partner"
     82 "retail-partner"
     71 "factory-outlet"
     32 "retail-store"
```

I think the first 2 are other stores that sell Asics stock. I'd be tempted to throw them away. I'd also set the `item["brand"]` to either "ASICS Factory Outlet" for `factory-outlet` or "ASICS" or for `retail-store`.

File: locations/spiders/asics_eu.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    allowed_domains = ["www.asics.com"]
+    start_urls = ["https://cdn.crobox.io/content/ujf067/stores.json"]
+
+    def parse(self, response):
+        for store in response.json():
+            if store["storetype"] in ["factory-outlet", "retail-store"]:
+                item = DictParser.parse(store)
+                item["street_address"] = item.pop("addr_full")
+                item["ref"] = store["name"]
+                item["brand"] = f'ascis-{store["storetype"]}'
```
Comment (iandees) [inline locations/spiders/asics_eu.py:19]:
```suggestion
                item["brand"] = f'asics-{store["storetype"]}'
```

This should be a human-readable name. Is the brand known as `asics-factory-outlet`? It probably should be "Asics Factory Outlet".

**Pull request merged**